test/JDBC/input/smalldatetimefromparts.sql

```diff
@@ -0,0 +1,94 @@
+SELECT SMALLDATETIMEFROMPARTS ( 1899, 12, 31, 23, 59 ) AS Result
```

Comment:
let us make sure that results match with SQL server

1) Try negative values for all args once
2) Try different input datatypes like Big int, real, text etc.
3) Try min/max date time if not already

Reply:
I checked every test case with sql server.

1. For each args the negative test cases are there.
2. Will Added
3. Those edge cases are there tried min and max for each args.